File: CS/ImplementCustomUserBasedOnExistingClassEF.Module/BusinessObjects/EmployeeTask.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl.EF;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ImplementCustomUserBasedOnExistingClassEF.Module.BusinessObjects {
    [DefaultClassOptions, ImageName("BO_Task")]
    public class EmployeeTask : Task {
        public EmployeeTask() : base() { }
        
        public virtual Employee Owner { get; set; }
    } 
}

File: VB/ImplementCustomUserBasedOnExistingClassEF.Module/BusinessObjects/EmployeeTask.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl.EF
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace ImplementCustomUserBasedOnExistingClassEF.Module.BusinessObjects
    <DefaultClassOptions, ImageName("BO_Task")> _
    Public Class EmployeeTask
        Inherits Task

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overridable Property Owner() As Employee
    End Class
End Namespace
